app/src/main/java/com/example/othregensburg/zapp/BardetailActivity.kt

```diff
@@ -0,0 +1,33 @@
+package com.example.othregensburg.zapp
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.widget.ImageView
+import android.widget.TextView
+import com.squareup.picasso.Picasso
+
+class BardetailActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_bardetail)
+
+        val navBarTitle = intent.getStringExtra(CustomViewHolder.BAR_TITLE_KEY)
+        supportActionBar?.title = navBarTitle
+
+
```

Comment:
There are some unnecessary blank lines.

Reply:
removed unnecessary  blank lines in new commit